### PR TITLE
Update self-hosted-runner-troubleshoot-snip.adoc

### PR DESCRIPTION
**Pull request by HennaAbbas**
# Description
A brief description of the changes.

# Reasons
A link to a GitHub and/or JIRA issue (if applicable).
Otherwise, a brief sentence about why you made these changes.

# Content Checklist
Please follow our style when contributing to CircleCI docs. Our style guide is here: [https://circleci.com/docs/style/style-guide-overview](https://circleci.com/docs/style/style-guide-overview).

Please take a moment to check through the following items when submitting your PR (this is just a guide so will not be relevant for all PRs) 😸:
updated command to check the logs for any of the previous runs
- [ ] Break up walls of text by adding paragraph breaks.
- [ ] Consider if the content could benefit from more structure, such as lists or tables, to make it easier to consume.
- [ ] Keep the title between 20 and 70 characters.
- [ ] Consider whether the content would benefit from more subsections (h2-h6 headings) to make it easier to consume.
- [ ] Check all headings h1-h6 are in sentence case (only first letter is capitalized).
- [ ] Is there a "Next steps" section at the end of the page giving the reader a clear path to what to read next?
- [ ] Include relevant backlinks to other CircleCI docs/pages.

### CODE REVIEW

**Review by christian-stephen — APPROVED** [review]

File: jekyll/_includes/snippets/troubleshoot/self-hosted-runner-troubleshoot-snip.adoc [hunk 1]
```diff
@@ -96,7 +96,8 @@ image::task-lifecycle.png[Image showing the task lifecycle with no content messa
 
 - **Pod restart:** Check if there were any container agent pod restarts around the time the workflow ran. If the pod was restarted around that time it would have resulted in the job not being processed. In such a case, we recommend rerunning the job once again.
 
-You can check the logs for any of the previous runs using the command `kubectl logs -n circleci <full pod name> --previous`.
+You can check the logs for any of the previous runs using the command `kubectl logs $(kubectl get pods -l app=container-agent -o jsonpath='{.items[0].metadata.name}' -n <namespace>) --previous -n <namespace>  >> container-runner.logs
+`.
```
Comment (christian-stephen) [inline jekyll/_includes/snippets/troubleshoot/self-hosted-runner-troubleshoot-snip.adoc:100]: Per internal discussion, we should probably just have this as the same command as before, but without the hardcoded `circleci` namespace.

**Review by christian-stephen — CHANGES_REQUESTED** [review]

File: jekyll/_includes/snippets/troubleshoot/self-hosted-runner-troubleshoot-snip.adoc (hunk 1 above)
Comment (rosieyohannan) [reply jekyll/_includes/snippets/troubleshoot/self-hosted-runner-troubleshoot-snip.adoc:100]: @christian-stephen so just `kubectl logs -n <namespace> <full pod name> --previous` ?

Comment (jrahme-cci) [reply jekyll/_includes/snippets/troubleshoot/self-hosted-runner-troubleshoot-snip.adoc:100]: Sorry it took so long to respond, but yes just as you put.

File: jekyll/_includes/snippets/troubleshoot/self-hosted-runner-troubleshoot-snip.adoc [hunk 2]
```diff
@@ -96,7 +96,8 @@ image::task-lifecycle.png[Image showing the task lifecycle with no content messa
 
 - **Pod restart:** Check if there were any container agent pod restarts around the time the workflow ran. If the pod was restarted around that time it would have resulted in the job not being processed. In such a case, we recommend rerunning the job once again.
 
-You can check the logs for any of the previous runs using the command `kubectl logs -n circleci <full pod name> --previous`.
+You can check the logs for any of the previous runs using the command `kubectl logs $(kubectl get pods -l app=container-agent -o jsonpath='{.items[0].metadata.name}' -n <namespace>) --previous -n <namespace>  >> container-runner.logs
```
Comment (rosieyohannan) [inline jekyll/_includes/snippets/troubleshoot/self-hosted-runner-troubleshoot-snip.adoc:99]:
```suggestion
You can check the logs for any of the previous runs using the command `kubectl logs  -n <namespace> <full pod name> --previous`
```

File: jekyll/_includes/snippets/troubleshoot/self-hosted-runner-troubleshoot-snip.adoc [hunk 3]
```diff
@@ -96,7 +96,8 @@ image::task-lifecycle.png[Image showing the task lifecycle with no content messa
 
 - **Pod restart:** Check if there were any container agent pod restarts around the time the workflow ran. If the pod was restarted around that time it would have resulted in the job not being processed. In such a case, we recommend rerunning the job once again.
 
-You can check the logs for any of the previous runs using the command `kubectl logs -n circleci <full pod name> --previous`.
+You can check the logs for any of the previous runs using the command `kubectl logs  -n <namespace> <full pod name> --previous`
```
Comment (rosieyohannan) [inline jekyll/_includes/snippets/troubleshoot/self-hosted-runner-troubleshoot-snip.adoc:99]:
```suggestion
You can check the logs for any of the previous runs using the command `kubectl logs -n <namespace> <full pod name> --previous`
```

**Review by rosieyohannan — APPROVED** [review]

**Review by christian-stephen — APPROVED** [review]

**Pull request merged**